content/source/docs/enterprise/sentinel/mock.html.md

```diff
@@ -0,0 +1,157 @@
+policies to operate on. Due to the highly variable structure of data that can be
+produced by an individual Terraform configuration, we provide you with the
+ability to generate mock data off of existing configurations. This can be of use
+when you need sample data for a new policy, or data to reproduce issues in an
+existing one.
+
+Testing policies is done using the [Sentinel
+Simulator](https://docs.hashicorp.com/sentinel/commands/). More general
+information on testing Sentinel policies can be found in the [Testing
+section](https://docs.hashicorp.com/sentinel/writing/testing) of the [Sentinel
+runtime documentation](https://docs.hashicorp.com/sentinel).
+
+~> **Be careful!** Mock data generated by Terraform Enterprise directly exposes
+any and all data within the configuration, plan, and state, including any
+sensitive data held within. Treat this data with care, and avoid generating
+mocks with live sensitive data if at all possible. To help secure access to this
+information, [write
+permission](/docs/enterprise/users-teams-organizations/permissions.html#write)
+on the workspace is necessary to generate mock data.
+
+## Generating Mock Data Using the UI
+
+Mock data can be generated using the UI by expanding the plan status section of
+any run page, and clicking on the **Download Sentinel mocks** button.
+
+![sentinel mock generate ui](/assets/images/guides/sentinel/download-mocks.png)
+
+For more information on creating a run, see [Running
+Terraform](/docs/enterprise/getting-started/runs.html) in the [Getting
+Started](/docs/enterprise/getting-started/index.html) guide.
+
+If the button is not visible, then the user or the plan is ineligible for mock
+generation. See [Mock Data Availability](#mock-data-availability) for more
+details.
+
+## Generating Mock Data Using the API
+
+Mock data is a feature of the [Plan Export
+API](/docs/enterprise/api/plan-exports.html).
+
+Multiple steps are required to perform mock generation. The export process is
+asynchronous, so monitoring the request is necessary to know when the data is
+available for download after requesting a generation.
+
+* Get the plan ID for the run that you want to generate the mock for. You can
+  see the plan ID for a run by [getting the run
+  details](/docs/enterprise/api/run.html#get-run-details).
+* [Request a plan
+  export](/docs/enterprise/api/plan-exports.html#create-a-plan-export) using the
+  discovered plan ID. You will want to supply a Sentinel export type, such as
+  `sentinel-mock-bundle-v0`.
+* Monitor the export request by [viewing the plan
+  export](/docs/enterprise/api/plan-exports.html#show-a-plan-export). When the
+  status has reached `finished`, the export data is ready for download.
+* Finally, you can [download the export
+  data](/docs/enterprise/api/plan-exports.html#download-exported-plan-data). You
+  have up to an hour to fetch the data after an export request has been made,
```

Comment:
```suggestion
  have up to an hour to fetch the data after an export request has been made.
```
Is it after request, or after generation, that the hour starts?

Reply:
After generation. I've clarified.